src/main/java/it/gov/pagopa/pu/bff/service/HealthIndicatorService.java

```diff
@@ -0,0 +1,62 @@
+package it.gov.pagopa.pu.bff.service;
+
+import it.gov.pagopa.pu.bff.config.MonitoringServiceConf;
+import it.gov.pagopa.pu.bff.model.ServiceStatus;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class HealthIndicatorService {
+
+  private final MonitoringServiceConf monitoringServiceConf;
+  private final RestTemplate restTemplate = new RestTemplate();
```

Comment:
lets build it using the injected RestTemplateBuilder in order to have default timeout and trace info